etc/test-helper.js

```diff
@@ -1,691 +1,4 @@
 /**
- * Initializes the module with the sync function at the specified file path.
```

Comment:
As I mentioned in the pull request description, the contents of this file have been moved verbatim to `src/test-helper.js`.